x-pack/plugins/monitoring/public/components/cluster/listing/listing.js

```diff
@@ -24,11 +24,11 @@ import {
 import { EuiMonitoringTable } from '../../table';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { i18n } from '@kbn/i18n';
-import { toMountPoint } from '../../../../../../../src/plugins/kibana_react/public';
 import { AlertsStatus } from '../../../alerts/status';
 import { STANDALONE_CLUSTER_CLUSTER_UUID } from '../../../../common/constants';
 import { getSafeForExternalLink } from '../../../lib/get_safe_for_external_link';
 import './listing.scss';
+import { toMountPoint, useKibana } from '../../../../../../../src/plugins/kibana_react/public';
```

Comment:
Just out of curiosity, does our bundler make a difference between relative vs. absolute import paths e.g.
```js
import { toMountPoint, useKibana } from 'src/plugins/kibana_react/public';
```
?

Reply:
Great question, I don't know the answer, but I relative linking does seem to be fairly common around the codebase.

Reply:
Nope, there is no real difference in absolute vs relative module resolution.

Reply:
Thank you both!